stock_mts_mto_mrp/README.rst

```diff
@@ -0,0 +1,92 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==================
+Stock MTS+MTO Rule
```

Comment:
Please, check the Reame and make improvements on how to use it, if there is a way to add a video, please do it.